case-lib/logging_ctl.sh

```diff
@@ -63,6 +63,22 @@ _func_log_directory()
     export LOG_ROOT=$log_dir/$timetag
 }
 
+# create dummy log files
+_func_log_dummy_files()
+{
+    LOG_LIST=([0]="slogger.txt" [1]="etrace.txt" [2]="dmesg.txt")
+    # do not create dummy files if three is no LOG_ROOT
+    if [ -n "$LOG_ROOT" ]; then
+        # create dummy files
+        for filename in "${LOG_LIST[@]}"
+        do
+            echo "dummy file $filename" > "$LOG_ROOT/$filename"
+        done
+    else
+        return
+    fi
+}
+
 for _func_ in $(declare -F|grep _func_log_|awk '{print $NF;}')
```

Comment:
check this for loop. It call all `_func_log_` functions when we source this file.